packages/endpoint-micropub/lib/utils.js

```diff
@@ -106,7 +107,7 @@ export const relativeMediaPath = (url, me) => url.includes(me) ? url.replace(me,
  * @param {string} timeZoneSetting Time zone setting
  * @returns {string} Path
  */
-export const renderPath = (path, properties, timeZoneSetting) => {
+export const renderPath = async (path, properties, publication, timeZoneSetting) => {
```

Comment:
Given that `timeZoneSetting` is a property of `publication`, I wonder if it’s worth only passing in `publication`, to reduce the number of parameters.

That said, that can be a later refinement/refactor.

Reply:
Makes sense to me. It was there already when I started. So I left it. My guess is that you'd likely want to refactor a handful of things if you want to go that route.